source/AI.cpp

```diff
@@ -2972,8 +2973,9 @@ void AI::DoScatter(Ship &ship, Command &command)
 		if(fabs(other->Acceleration() / acceleration - 1.) > .05)
 			continue;
 
-		// Move away from this ship. What side of me is it on?
-		command.SetTurn(offset.Cross(ship.Facing().Unit()) > 0. ? 1. : -1.);
+		// We are too close to this ship. Turn away from it if we aren't already facing away.
+		if(fabs(other->Facing().Unit().Dot(ship.Facing().Unit())) > 0.99) // 0.99 => 8 degrees
```

Comment:
Why take the magnitude of the dot product?

Reply:
It handles the case where the ships are facing directly away from each other. Such oddities can happen if you tell a large number of escorts to gather around you. Or, if you tell them all to stop somewhere, and then leave that spot. I know it is a corner case, but this entire bug is one big corner case.